### ISSUE TITLE
Automatically go to home after a minute of inactivity

### ISSUE THREAD

**Issue by chagai95** [vector-im/element-web#19923]
#### What would you like to do?
Automatically go to home after a minute of inactivity.

#### Why would you like to do it?
To avoid missing messages from people when you forget the tab open on their chat

#### How would you like to achieve it?
Move to home screen

### Have you considered any alternatives?

Mark it as unread only when clicking on the chat again, I think there is already done mechanism for this already but it seems to not work for me sometimes.

### Additional context

It would be cool to promote the home button more in general,  otherwise you barely ever see it.

**Comment by ShadowJonathan** [comment]
Make it an hour, and configurable, and I'd be happy with this.

Although, the home space could also be open on a room, and switching to it could be auto-reading *that* room instead.

**Comment by chagai95** [comment]
Ah no I meant the home under #home not the home space...

**Comment by SimonBrandner** [comment]
> Make it an hour, and configurable, and I'd be happy with this.

Yes, if this were to be a feature, it would need to configurable

**Comment by dbkr** [comment]
> To avoid missing messages from people when you forget the tab open on their chat

This makes me feel like this feature would be a workaround for something that's not working correctly. If having the room open prevents the messages from notifying sensibly, fixing that would seem like the better option?

**Comment by aaronraimist** [comment]
Related: https://github.com/vector-im/element-web/issues/5057

**Comment by bkil** [comment]
For precisely this I have 1 room in each of my account that I never read marked as favorite and I always go back to those rooms before tabbing away! Another use case is if you switch off your computer daily and reload all tabs where you left off, you will be otherwise "forced" to start chatting on the last room no matter what. Otherwise you risk losing your unread status in the room where the browser started. If this is a room with a heavy traffic (perhaps even off topic, or otherwise unimportant), you will start your day with half an hour of scrolling through before you can start to communicate or react in rooms that really matter.

I think in general, a mechanism for marking rooms/conversations unread, starring them for later review or something like that would be a really useful feature.
